docs/installation/linux/fedora.md

```diff
@@ -94,36 +110,57 @@ There are two ways to install Docker Engine.  You can install with the `dnf` pac
         To try something more ambitious, you can run an Ubuntu container with:
          $ docker run -it ubuntu bash
 
+        Share images, automate workflows, and more with a free Docker Hub account:
+         https://hub.docker.com
+
         For more examples and ideas, visit:
-         http://docs.docker.com/userguide/
+         https://docs.docker.com/engine/userguide/
 
+If you need to add an HTTP Proxy, set a different directory or partition for the
+Docker runtime files, or make other customizations, read our Systemd article to
+learn how to [customize your Systemd Docker daemon options](../../admin/systemd.md).
 
 ### Install with the script
 
+You use the same installation procedure for all versions of Fedora.
 
 1. Log into your machine as a user with `sudo` or `root` privileges.
 
-2. Make sure your existing dnf packages are up-to-date.
+2. Make sure your existing packages are up-to-date.
 
-		$ sudo dnf update
+    ```bash
+    $ sudo dnf update
+    ```
 
 3. Run the Docker installation script.
 
-		$ curl -fsSL https://get.docker.com/ | sh
+    ```bash
+    $ curl -fsSL https://get.docker.com/ | sh
+    ```
 
-	This script adds the `docker.repo` repository and installs Docker.
+    This script adds the `docker.repo` repository and installs Docker.
 
 4. Enable the service.
 
-		$ sudo systemctl enable docker.service
+    ```bash
+    $ sudo systemctl enable docker.service
+    ```
 
 5. Start the Docker daemon.
 
-        $ sudo systemctl start docker
+    ```bash
+    $ sudo systemctl start docker
+    ```
 
 6. Verify `docker` is installed correctly by running a test image in a container.
 
-		$ sudo docker run hello-world
+    ```bash
+    $ sudo docker run hello-world
+    ```
+
+If you need to add an HTTP Proxy, set a different directory or partition for the
+Docker runtime files, or make other customizations, read our Systemd article to
+learn how to [customize your Systemd Docker daemon options](../../admin/systemd.md).
 
 ## Create a docker group
```

Comment:
We try to avoid using `code` blocks in headings (therefore left out here), also, `docker` in this case is not a command, but the name of the group